grassroots-frontend/src/routes/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { createFileRoute } from '@tanstack/react-router'
+
+export const Route = createFileRoute('/')({
+  component: Index,
+})
+
+function Index() {
+  return (
+    <p>TODO</p>
+  )
+}
```

Comment:
Based on the file endings, I think this might not have been prettified

Reply:
Done!